bears/setup.py.in

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+import locale
+
+import setuptools.command.build_py
+from setuptools import find_packages, setup
+
+try:
+    locale.getlocale()
+except (ValueError, UnicodeError):
+    locale.setlocale(locale.LC_ALL, 'en_US.UTF-8')
+
+
+if __name__ == "__main__":
+    setup(name='$NAME',
```

Comment:
hm okay with `$NAME` it doesn't work probably as you use the value again later^^